app/portainer/hostmanagement/fdo/fdo.service.ts

```diff
@@ -23,13 +23,66 @@ export async function configureDevice(
   }
 }
 
+export async function createProfile(
+  profileName: string,
+  method: string,
+  profileFileContent: string
+) {
+  const payload: Record<string, unknown> = {
```

Comment:
```suggestion
  const payload = {
```

don't need the type as typescript is smart enough to identify it

Reply:
done